core.js

```diff
@@ -2174,6 +2174,10 @@ class HTMLScriptElement extends HTMLLoadableElement {
     this.innerHTML = textContent;
   }
 
+  get textContent() {
```

Comment:
I don't think this is what the spec says. If there is a missing accessor it should probable be on some parent class, not Script.